server/models/articles.js

```diff
@@ -83,6 +83,17 @@ const getArticleModel = (sequelize, DataTypes) => {
       target: "id",
       onDelete: "CASCADE"
     });
+
+    Article.belongsToMany(db.Tags, {
+      through: {
+        model: "ArticleTags",
+        unique: false
+      },
+      timestamps: false,
+      foreignKey: "articleId",
+      target: "id",
+      onDelete: "RESTRICT"
```

Comment:
Isn't it restricted by default 🤔

Reply:
For n:m association, the default for onDelete is CASCADE